SKILL.tsv

```diff
@@ -1506,7 +1506,7 @@ SKILL_20150317_001505	$Statue of Goddess Laima
 SKILL_20150317_001506	{memo X}Carve Goddess Zemyna Statue. Spalsh of party members around the statue is increased.
 SKILL_20150317_001507	{memo X}Splash +#{CaptionRatio}# {nl}Statue duration 30 seconds {nl} Use 5 cedar wood
 SKILL_20150317_001508	$Carve
-SKILL_20150317_001509	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Use carving knife to attack the target. Additional damage applies to plant type monsters and can obtain statue materials
+SKILL_20150317_001509	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Use a carving knife to attack. Plant type monsters take additional damage and may drop statue materials.
```

Comment:
Use a carving knife to attack -> Attack the target by using a carving knife.

Plant type monsters take additional damage and may drop statue materials. -> Plant-type monsters take additional bonus damage and may drop statue materials.